From 478036b92d4f633189dbbfa24ceee536c5df92da Mon Sep 17 00:00:00 2001
From: Zhao Zhili <zhilizhao@tencent.com>
Date: Mon, 18 Nov 2024 16:43:28 +0800
Subject: [PATCH 33/34] avfilter/textutils: Add missing time_internal.h

This fix build error on Windows with MSVC. We need time_internal.h
for gmtime_r and localtime_r.

Signed-off-by: Zhao Zhili <zhilizhao@tencent.com>
---
 libavfilter/textutils.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libavfilter/textutils.c b/libavfilter/textutils.c
index e6b5239b20..eabf842ab5 100644
--- a/libavfilter/textutils.c
+++ b/libavfilter/textutils.c
@@ -31,6 +31,7 @@
 #include "libavutil/file.h"
 #include "libavutil/mem.h"
 #include "libavutil/time.h"
+#include "libavutil/time_internal.h"
 
 static int ff_expand_text_function_internal(FFExpandTextContext *expand_text, AVBPrint *bp,
                                             char *name, unsigned argc, char **argv)
-- 
2.39.5 (Apple Git-154)

